components/org.wso2.carbon.identity.media.endpoint/src/main/java/org/wso2/carbon/identity/media/endpoint/exception/mapper/JsonProcessingExceptionMapper.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.media.endpoint.exception.mapper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.media.endpoint.Error;
+import org.wso2.carbon.identity.media.endpoint.common.MediaServiceConstants;
+
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import static org.wso2.carbon.identity.media.endpoint.common.Util.getError;
+
+/**
+ * Handles exceptions when an incorrect json requests body is received.
+ * Sends a default error response.
+ */
+public class JsonProcessingExceptionMapper implements ExceptionMapper<JsonProcessingException> {
+
+    private static final Log log = LogFactory.getLog(JsonProcessingExceptionMapper.class);
+
+    @Override
+    public Response toResponse(JsonProcessingException e) {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Provided JSON request content is not in the valid format:", e);
```

Comment:
remove ":" at the end

Reply:
fixed with: b04c81c84c80a9eef1a43eb56605b6224a3cb7cc